Lake/CLI/Main.lean

```diff
@@ -192,6 +192,10 @@ def verifyInstall : CliStateM PUnit := do
 /-- Exit code to return if `print-paths` cannot find the config file. -/
 def noConfigFileCode : ExitCode := 2
 
+/-- Environment variable that is set when `lake serve` cannot parse the lakefile
+and falls back to plain `lean --server`. -/
+def cannotParseLakefileEnv := "LAKE_INTERNAL_CANNOT_PARSE_LAKEFILE"
```

Comment:
```suggestion
def invalidConfigEnv := "LAKE_INVALID_CONFIG"
```

Shorter and more general. I would also suggest just inling it (especially since all other environment variables are inlined).

Reply:
I'm not sure if more general is a good thing, this is very much an internal implementation detail and shouldn't be used outside of Lake.  But I don't have a strong opinion on the name.

I've left the definition because it seems like a good place to document the purpose of the environment variable.